src/frontend/login/myStyle.css

```diff
@@ -0,0 +1,45 @@
+body {
+    color: white;
+    font-family: "Lucida Console", Monaco, monospace;
+    font-size: 24px;
+    /* text-shadow: 4px 4px 4px #aaa; */
+    background-image: url("stars3.gif");
+    background-size: cover;
+    background-position: center center;
+    background-repeat: repeat;
+    background-size: 100% 100%;
+    width: 100%;
+    height: 100%;
+    min-height: 100%;
+}
```

Comment:
styling is good, but creates an annoying white background where the div is. can you fix this? (should just be able to remove the background-color: white in the style:
https://github.com/Fisherswamp/sfarts/blob/35bb55e8f42701d70f91382ac8895df20738ccb2/src/frontend/FrontendAppComponent.tsx#L83

Reply:
Please test it though